plugins/modules/ndo_route_map_policy_route_control_context.py

```diff
@@ -0,0 +1,434 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Sabari Jaganathan (@sajagana) <sajagana@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_route_map_policy_route_control_context
+short_description: Manage Route Map Policy for Route Control Context on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Route Map Policy for Route Control Context on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v4.1 and later.
+author:
+- Sabari Jaganathan (@sajagana)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be a tenant template.
+    - This parameter or O(template_id) is required.
+    type: str
+  template_id:
+    description:
+    - The ID of the tenant template.
+    - This parameter or O(template) is required.
+    type: str
+  route_map_policy:
+    description:
+    - The name of the Route Map Policy for Route Control.
+    - This parameter or O(route_map_policy_uuid) is required.
+    type: str
+  route_map_policy_uuid:
+    description:
+    - The UUID of the Route Map Policy for Route Control.
+    - This parameter or O(route_map_policy) is required.
+    type: str
+  name:
+    description:
+    - The name of the Route Control context.
+    type: str
+  description:
+    description:
+    - The description of the Route Control context.
+    - Providing an empty string O(description="") will remove description from the Route Control context.
+    type: str
+  order:
```

Comment:
Does order have a default or is it required? It looks required in the UI and also the UI says 0-9?

Reply:
Updated! Also order is marked as a required field in the UI, but it defaults to 0 when we create the object using API.